agenta-web/src/components/Playground/Views/ParametersView.tsx

```diff
@@ -217,6 +231,15 @@ const ParametersView: React.FC<Props> = ({
                                                 onChange={(value) => onChange(param, value)}
                                             />
                                         )}
+                                        {param.type === "integer" && (
+                                            <InputNumber
+                                                min={0}
```

Comment:
we should be taking the min and max from the api, no?
param.minimum and param.maximum